### ISSUE TITLE
api_conform_test failing with Dart roll

### ISSUE THREAD

**Issue by rmacnak-google** [flutter/flutter#185423]
```
Checking the integrity of the Web SDK
Running /b/s/w/ir/cache/builder/engine/src/flutter/web_sdk/test/api_conform_test.dart
Checking 111 public classes.
Warning: lib/ui/ui.dart SemanticsUpdateBuilder.updateNode parameter 10 traversalParent has a different nullability than in lib/web_ui/ui.dart.
Warning: lib/ui/ui.dart SemanticsUpdateBuilder.updateNode parameter 26 tooltip has a different nullability than in lib/web_ui/ui.dart.
```
[log](https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8683813467001638225/+/u/test:_analyze_dart_ui/stdout)

It looks like the nullability of some parameters is computed differently. Looking at the source, the two definitions do in fact disagree.

**Comment by loic-sharma** [comment]
It looks like native requires non-null values, but web accepts null values. I suspect we should update web to also require non-null values. cc @chunhtai 

Native:

https://github.com/flutter/flutter/blob/5e4f1693184722bee16330188ab4e0d8e1f9cefc/engine/src/flutter/lib/ui/semantics.dart#L1936-L1980

Web:

https://github.com/flutter/flutter/blob/5e4f1693184722bee16330188ab4e0d8e1f9cefc/engine/src/flutter/lib/web_ui/lib/semantics.dart#L714-L758

**Comment by chunhtai** [comment]
Is the test not part of presubmit?
